IOS_SETUP.md

```diff
@@ -32,6 +32,28 @@ Additionally, you must configure the background task identifiers. The default id
 </plist>
 ```
 
+
+You MUST amend your `AppDelegate.swift` and `Info.plist` file to register your task ID.
+
+- AppDelegate.swift
+```
+WorkmanagerPlugin.registerTask(withIdentifier: "task-identifier")
+```
+
+- Info.plist
+```
+<key>BGTaskSchedulerPermittedIdentifiers</key>
+	<array>
+		<string>task-identifier</string>
+  </array>
+</key>
+```
+
+**NOTE: **
+- Every custom task ID has to be registered as above
+- Please refer the sample app for reference
+- The task identifier should be prefixed with app bundle id
```

Comment:
If that is the case, please can you adjust the strings in the example as well (and link to the docs suggesting the prefix)